sig-scheduling/charter.md

```diff
@@ -0,0 +1,67 @@
+# SIG Scheduling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+We build Kubernetes schedulers and scheduling features for Pods. We design and
+implement features that allows users to customize placement of Pods on the nodes
+of a cluster. These features include those that improve reliability of workloads,
+more efficient use of cluster resources, and/or enforces placement policies. 
+
+### In scope
+
```

Comment:
nit: please link to the README.md file here as well for the sig.  i think the template had said the sigs.yaml but i have yet to see a stable link work there.

https://github.com/kubernetes/community/blob/master/sig-node/charter.md#in-scope

Reply:
gives an example that had settled on with brian.

Reply:
done.